website/docs/docs/dbt-cloud/setting-up-dbt-cloud/create-an-environment/development-environment.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Create a new dbt Cloud Project"
```

Comment:
the titles here will change, right? to be more like "Set up a development environment"?